contrib/long_distance_matching/ldm.h

```diff
@@ -0,0 +1,55 @@
+#ifndef LDM_H
```

Comment:
You only need one copy of the header, since they expose the same set of functions.